library/include/rocblas-types.h

```diff
@@ -28,8 +28,11 @@ typedef int32_t rocblas_int;
 typedef float2  rocblas_float_complex;
 typedef double2 rocblas_double_complex;
 // half type TODO put name of half here
-typedef float    rocblas_half;
+typedef __fp16   rocblas_half;
 typedef float2   rocblas_half_complex;
+// half vectors
+typedef __fp16 half8 __attribute__((ext_vector_type(8)));
+typedef __fp16 half2 __attribute__((ext_vector_type(2)));
```

Comment:
I recommend not including hip_fp16.h at all in rocblas as of now. HIP half2 will not generate vector code.